app/components/Transfers/configureDelegation/DelegationAmountPage.tsx

```diff
@@ -184,9 +184,11 @@ export default function DelegationAmountPage({
             <div className="flexChildFill">
                 {existing !== undefined || (
                     <p className="mT0">
-                        This transaction will delegate an amount of CCD to an
-                        active baker. You must choose the amount to delegate, if
-                        you want to add rewards to the delegated amount.
+                        This transaction will delegate an amount of CCD to{' '}
+                        {target === null
+                            ? 'the L-pool'
```

Comment:
It is no longer called the L-pool, so we have to take that into consideration here.

Reply:
Yes, I assume this will be corrected when the document describing the changes needed is in place.. It hasn't been corrected the other places either, has it?